server/data/data-postgres/src/main/java/org/apache/james/rrt/postgres/PostgresRecipientRewriteTableModule.java

```diff
@@ -45,7 +45,7 @@ interface PostgresRecipientRewriteTableTable {
                 .column(DOMAIN_NAME)
                 .column(TARGET_ADDRESS)
                 .constraint(DSL.constraint(PK_CONSTRAINT_NAME).primaryKey(USERNAME, DOMAIN_NAME, TARGET_ADDRESS))))
-            .supportsRowLevelSecurity();
+            .disableRowLevelSecurity();
```

Comment:
Why as well?

Reply:
Idem, outside of the mailbox. rls do not make sense on an email flux.